packages/webapp/src/components/AddCropVariety/index.jsx

```diff
@@ -81,24 +90,99 @@ export default function PureAddCropVariety({
         value={progress}
       />
 
-      <div className={styles.cropLabel}>{cropNameLabel}</div>
-      <img
-        src={crop_variety_photo_url}
-        alt={crop.crop_common_name}
-        className={styles.circleImg}
-        onError={(e) => {
-          e.target.onerror = null;
-          e.target.src = 'crop-images/default.jpg';
-        }}
-      />
+      <p style={{ fontSize: '16px', marginBottom: '20px' }}>
+        {t('translation:CROP.VARIETAL_SUBTITLE')}
+      </p>
 
+      <div style={{ display: 'flex', position: 'relative' }}>
+        <div style={{ width: 'fit-contents', display: 'inline-block' }}>
+          <img
+            src={crop.crop_photo_url}
+            alt={crop.crop_common_name}
+            className={styles.circleImg}
+            onError={(e) => {
+              e.target.onerror = null;
+              e.target.src = 'crop-images/default.jpg';
+            }}
+          />
+        </div>
+        <div
+          className="nameLabel"
+          style={{
+            display: 'inline-block',
+            position: 'absolute',
+            top: '48%',
+            transform: 'translateY(-50%)',
+            left: '140px',
+          }}
+        >
+          <p
+            className="cropLabel"
+            style={{ fontWeight: '600', fontSize: '18px', marginBottom: '0px' }}
+          >
+            {cropNameLabel}
+          </p>
+          <p
+            className="scientificNameLabel"
+            style={{ verticalAlign: 'top', fontStyle: 'italic', fontSize: '16px', height: '18px' }}
+          >
+            {scientificNameLabel}
+          </p>
+        </div>
+      </div>
+      <Input
+        data-cy="crop-variety"
+        style={{ marginBottom: '40px', marginTop: '40px' }}
+        label={t('CROP.VARIETY_COMMON_NAME')}
+        type="text"
+        hookFormRegister={commonNameRegister}
+        hasLeaf={false}
+      />
+      <Input
+        data-cy="crop-varietal"
+        style={{ marginBottom: '40px' }}
+        label={t('CROP.VARIETY_VARIETAL')}
+        type="text"
+        hookFormRegister={varietalRegister}
+        errors={getInputErrors(errors, 'crop_varietal')}
+        hasLeaf={false}
+        optional
+        subText={t('CROP.VARIETAL_SUBTEXT')}
+        link={'https://www.litefarm.org/post/cultivars-and-varietals'}
+        placeholder={t('CROP.VARIETAL_PLACEHOLDER')}
+      />
+      <Input
+        data-cy="crop-cultivar"
+        style={{ marginBottom: '40px' }}
+        label={t('CROP.VARIETY_CULTIVAR')}
+        type="text"
+        hookFormRegister={cultivarRegister}
+        errors={getInputErrors(errors, 'crop_cultivar')}
+        hasLeaf={false}
+        optional
+        subText={t('CROP.CULTIVAR_SUBTEXT')}
+        link={'https://www.litefarm.org/post/cultivars-and-varietals'}
+        placeholder={t('CROP.CULTIVAR_PLACEHOLDER')}
+        hideOnFocus
```

Comment:
@MwayaB 

Getting an error on this line: 
![image](https://user-images.githubusercontent.com/30075973/223565612-57da08a9-9c4d-4ea0-a581-1f31b10e2df5.png)

Reply:
Yes, that happens because of this bit of code `input:focus::placeholder {
  color: transparent;
}` . Forgot to delete that bit of code